R/data_import.R

```diff
@@ -116,8 +116,8 @@ import_data <- function(dataset = NULL,
     } else stop("File type not recognised")
 
   # Create preparation template
-  qtemplate(
-    "qData-prep.R",
+  manytemplate(
+    "package-prep.R",
```

Comment:
If most files start with an upper case  "P", please make sure all files follow for consistency.